web-app/app/assets/stylesheets/base/_buttons.scss

```diff
@@ -0,0 +1,23 @@
+.basic-button {
+  background-color: $light-blue;
+  color: $white;
+  font-size: $button-font-size;
+  font-weight: bold;
+  padding: 20px 60px;
+  border-radius: 10px;
+  border: none;
+  cursor: pointer;
+  &:hover, &:focus {
+    background-color: $light-blue2;
```

Comment:
Actually I'll create a variable for colors and hovers in buttons :-) I'll update and let you know.